package.json

```diff
@@ -38,7 +38,7 @@
     "@edx/frontend-platform": "8.1.5",
```

Comment:
>[!IMPORTANT]
> Needs to be updated to a version compatible with `paragon 23`

Reply:
Moved this message to the PR description, thanks!